docs/source/array-random.rst

```diff
@@ -1,5 +1,5 @@
-dask.array.random
-=================
+Random Arrays
+=============
 
 ``dask.array`` copies the ``numpy.random`` module for all univariate
 distributions.  The interface to each function is identical except for the
```

Comment:
I had the same thought about this page (consolidating into API docs). Could you handle this or suggest the best way to do this? My Sphinx knowledge here is limited.